pom.xml

```diff
@@ -0,0 +1,205 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.api</groupId>
+  <artifactId>gax-parent</artifactId>
+  <packaging>pom</packaging>
+  <version>2.7.1-SNAPSHOT</version><!-- {x-version-update:gax:current} -->
+  <name>GAX (Google Api eXtensions) for Java (Parent)</name>
+  <url>https://github.com/googleapis/gax-java</url>
+  <description>Google Api eXtensions for Java (Parent)</description>
+
+  <parent>
+    <groupId>com.google.cloud</groupId>
+    <artifactId>google-cloud-shared-config</artifactId>
+    <version>1.2.2</version>
+  </parent>
+
+  <developers>
+    <developer>
+      <id>GoogleAPIs</id>
+      <name>GoogleAPIs</name>
+      <email>googleapis@googlegroups.com</email>
+      <organization>Google LLC</organization>
+      <organizationUrl>https://www.google.com</organizationUrl>
+    </developer>
+  </developers>
+
+  <scm>
+    <connection>scm:git:https://github.com/googleapis/gax-java.git</connection>
+    <url>https://github.com/googleapis/gax-java</url>
+  </scm>
+
+  <licenses>
+    <license>
+      <name>BSD</name>
+      <url>https://github.com/googleapis/gax-java/blob/master/LICENSE</url>
+    </license>
+  </licenses>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+
+  <modules>
+    <module>gax</module>
+    <module>gax-grpc</module>
+    <module>gax-httpjson</module>
+    <module>gax-bom</module>
```

Comment:
This may be problematic? The `gax-bom/pom.xml` is a pre-written file for BOM publication, not to set up a project. Should we rename it to `gax-bom/Bom.xml` or `pom-for-bom.xml` and create a new project BOM? Or, is it that Maven publication will copy this file as-is when publishing a BOM?

Reply:
I'm not sure I follow. I'm just including it here so that it's part of the multi-module project. It doesn't use `gax-parent` as the parent. So, it can be released independently.

Reply:
Let me ask this way: the current behavior about the BOM publishing is that, the file `gax-bom/pom.xml` is used as-is for the BOM. That is, the [content of the published BOM here](https://search.maven.org/artifact/com.google.api/gax-bom/2.7.0/pom) is precisely `gax-bom/pom.xml`. So, I'm trying to make sure that, after migrating to Maven, this behavior doesn't change.

Reply:
Why would it change? Notice that I didn't add `gax-parent` to `gax-bom`.